docs/source/governance.rst

```diff
@@ -1,12 +1,12 @@
 .. _governance:
 
-PyTorch Lightning Governance | Persons of interest
-==================================================
+Lightning Governance | Persons of interest
+==========================================
```

Comment:
Why delete? The name was accurate or not?

Reply:
it was too long in the left menu

Reply:
But we have multiple Lightning projects now 😄